ozaria/site/views/play/level/PlayLevelView.js

```diff
@@ -1394,13 +1419,23 @@ class PlayLevelView extends RootView {
     if (this.destroyed) {
       return
     }
+
     Backbone.Mediator.publish('level:set-time', { ratio: 1 })
+
+    // Don't award achievements when beating level changed in level editor
     if (this.level.hasLocalChanges()) {
       return
-    } // Don't award achievements when beating level changed in level editor
-    if (this.goalManager.checkOverallStatus() === 'success') {
+    }
+
+    const options = { showModal: true }
+    const state = this.session.get('state')
+
+    if (this.goalManager.finishLevel()) {
+      if (state && state.capstoneStage && this.goalManager.getRemainingGoals().length === 0) {
+        options.capstoneInProgress = true
```

Comment:
I believe setting this would still not set `complete: true` in the overall state in `LevelBus` `onVictory` method - is that right?